stdlib/3/tempfile.pyi

```diff
@@ -17,6 +18,11 @@ TMP_MAX: int
 tempdir: Optional[str]
 template: str
 
+if sys.version_info >= (3, 6):
+    _DirT = Union[AnyStr, os.PathLike]
```

Comment:
Haven't tested, but should this be `Union[AnyStr, os.PathLike[AnyStr]]`?

I vaguely remember a mypy bug around this.

Reply:
ok! let's try that :D

Reply:
seems still not happy :thinking: